### PR TITLE
feat(def): add built-in workflow definitions

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:
add built-in workflow definitions

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/cc @hongchaodeng

### CODE REVIEW

File: charts/vela-core/templates/defwithtemplate/apply-component.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Definition source cue file: vela-templates/definitions/internal/apply-component.cue
+apiVersion: core.oam.dev/v1beta1
+kind: WorkflowStepDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: Apply components and traits for your workflow steps
+  name: apply-component
+  namespace: {{.Values.systemDefinitionNamespace}}
+spec:
+  podDisruptive: false
```
Comment (wonderflow) [inline charts/vela-core/templates/defwithtemplate/apply-component.yaml:11]: workflow step def 里面没这个字段

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -164,25 +164,11 @@ jobs:
...
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GO_VERSION }}
-        id: go
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-      - name: Run cross-build
-        run: make cross-build
       - name: Run compress binary
         run: make compress
```
Comment (wonderflow) [inline .github/workflows/go.yml:174]: 删掉？

File: .github/workflows/go.yml [hunk 3]
```diff
@@ -164,25 +164,11 @@ jobs:
...
-      - name: Run make reviewable
-        run: make reviewable
```
Comment (wonderflow) [inline .github/workflows/go.yml:188]: 最后要加个步骤清理环境

File: .github/workflows/go.yml (hunk 2 above)
Comment (FogDong) [reply .github/workflows/go.yml:174]: 加在前面了

**Review by wonderflow — APPROVED** [review]

**Pull request merged**